onnxruntime/test/onnx/main.cc

```diff
@@ -635,7 +635,64 @@ select from 'TF8', 'TF16', 'UINT8', 'FLOAT', 'ITENSOR'. \n)");
             ORT_TSTR("operator_pow"),
             ORT_TSTR("bernoulli"),
             ORT_TSTR("bernoulli_double"),
-            ORT_TSTR("bernoulli_seed")};
+            ORT_TSTR("bernoulli_expanded"),
```

Comment:
this file has too many big lists...

at the top of this one:
```
    // TODO: Get these from onnx_backend_test_series_filters.jsonc.
    // Permanently exclude following tests because ORT support only opset staring from 7,
    // Please make no more changes to the list
    static const ORTCHAR_T* immutable_broken_tests[] =
```

so if there are new tests to disable for now, due to lack of float8 support, they probably don't belong in this one. I suppose we can add a new list just for this case, e.g., `float8_disabled_tests`

Reply:
It should not be needed. I'll try to remove them from the list.